tests/queries/0_stateless/01375_output_format_with_names.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. "$CURDIR"/../shell_config.sh
+
+opts=(
+    --input-format CSV
+    -S 'key String'
+    -q 'SELECT key FROM table'
+)
+
+function produce()
+{
+    echo 1
+    echo 1
+}
+
+echo 'TSVWithNames'
+produce | ${CLICKHOUSE_LOCAL} "${opts[@]}" --format TSVWithNames
```

Comment:
Is the `produce` part important, or `select '1' key from numbers(2) format TSVWithNames ` would also work?

Reply:
egh, was too enthusiastic about scripting, will use your suggestion